src/en/users-creating.md

```diff
@@ -83,8 +83,11 @@ juju login
 ```
 
 Once a user logs in they become the current user. The following is a quick way
-to determine the current user:
+to determine the current user (as well as the current controller and model):
 
 ```bash
-juju show-user
+juju whoami
 ```
+
+The command `juju show-user` can also be used to get information on the current
```

Comment:
Command `show-user` *does display* what the current user is, along with some other information on the user. So it **also** gives the current user *in addition* to other information. So maybe:

"The command `juju show-user` can also be used to get the current user, in addition to other information on the user."